.circleci/config.yml

```diff
@@ -1083,13 +1083,6 @@ jobs:
             JEST_JUNIT_OUTPUT_DIR: ./reports/
       - store_test_results:
           path: libs/ui/reports/
-      - when:
-          condition:
-            equal: [ main, << pipeline.git.branch >> ]
-          steps:
-            - run:
-                name: Export App Strings Catalog
-                command: echo 0 # TODO(kofi): export to versioned folder accessible from VxDesign
```

Comment:
The hypothetical versioned folder accessible from VxDesign mentioned here is now just `libs/ui` itself, specifically `libs/ui/src/ui_strings/app_strings_catalog/`